doc/LETS_ENCRYPT_CONFIG.md

```diff
@@ -0,0 +1,49 @@
+Here are the steps to get Let's Encrypt SSL certificates on Chef server
+
+##Prerequisites
+
+1. Chef server must be reachable on port 80 or 443 on a public IP address
```

Comment:
Oh I see. So to get a proper TLS cert, you'd have to expose a non-HTTPS port for the challenge mechanism?

If so, I think we might have to change some more of that config... but you're trying this out as far as I've understood, so you will find a way... 🤔